resources/css/navigation.css

```diff
@@ -0,0 +1,27 @@
+@tailwind base;
+ 
+@tailwind components;
+ 
+@tailwind utilities;
+
+.header{
+	
+}
+.siteLogo{
+	max-width: 100px;
+}
+
+.navigationBar{
+	font: 15px Arial, sans-serif;
+	vertical-align: top;
+}
+.navItems{
+	background-color: white;
+	border: none;
+	color: blue;
+	padding: 15px 15px;
+	text-decoration: none;
+	display: inline-block;
+	font-size: 16px;
+
```

Comment:
Style: remove the extra empty line in the class

Reply:
removed